.travis.yml

```diff
@@ -0,0 +1,22 @@
+sudo: required
+dist: xenial
+language: generic
+services:
+  - docker
+
+notifications:
+  email:
+    on_failure: always
+    recipients:
+      - ros@pilz.de
+env:
+  global:
+    - CATKIN_PARALLEL_TEST_JOBS=-p1
+    - ROS_PARALLEL_TEST_JOBS=-j1
+  matrix:
+    - ROS_DISTRO="kinetic" ROS_REPO=ros
+    - ROS_DISTRO="kinetic" ROS_REPO=ros-shadow-fixed
+install:
+  - git clone https://github.com/ros-industrial/industrial_ci.git .industrial_ci
```

Comment:
please use `--depth 1` to fetch only the most recent state of the git repository